vision-zero

```diff
@@ -94,11 +96,16 @@ def replicateDb(args):
     graphqlEngineDown(args)
 
     # print(tool_runner_command + replicate_command)
-    today = date.today().strftime("%Y-%m-%d")
-    snapshotFilename = (
-        "visionzero_" + today + "_" + (
-            "with-change-log" if args.include_change_log_data else "without-change-log") + ".sql"
-    )
+    snapshotFilename = ""
+    if(args.filename):
+        snapshotFilename = args.filename
+    else:
+        today = date.today().strftime("%Y-%m-%d")
+        snapshotFilename = (
+            "visionzero_" + today + "_" + (
+                "with-change-log" if args.include_change_log_data else "without-change-log") + ".sql"
+        )
```

Comment:
I reallllly like that you can specify a filename to be used when pulling data down from the production machine.